thorlcr/activities/keyedjoin/thkeyedjoinslave.cpp

```diff
@@ -2077,6 +2106,8 @@ class CKeyedJoinSlave : public CSlaveActivity, implements IJoinProcessor, implem
             PARENT::end();
             doClose(kjs_fetchclose);
         }
+        // Not tracking remote fetch file stats.  Future: may be something to consider.
```

Comment:
can you open a JIRA and link it to this JIRA ?

Reply:
Will do. TODO: create ticket once jira is working again.

Reply:
jira: https://track.hpccsystems.com/browse/HPCC-31000